server/service/FlightService.js

```diff
@@ -340,7 +341,10 @@ const flightService = {
     return updatedColumns;
   },
 
-  delete: async (flight) => {
+  delete: async (flight, { message }) => {
```

Comment:
Shouldn't there be a "rejectFlight" service to separate from regular deleting?
If I delete a flight with a not accepted violation elsewhere it will send an empty email.

Reply:
Yep... your right and I modified it